system-probe_x64/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM debian:bullseye
```

Comment:
We might want to base this on an older distro to not require a too new glibc.

Reply:
The thing is that I already wasted a lot of time trying to build `iovisor/bcc` on an old distro and it was a big fail.
The good news is that there’s already safety belts that are guaranteeing that this is not an issue.
Indeed, what happens in DataDog/datadog-agent#4619 is that:
* `iovisor/bcc` is built inside this `system-probe_$ARCH` docker image environment;
* `system-probe` is built inside this `system-probe_$ARCH` docker image environment (because `iovisor/gobpf` also requires up-to-date glibc and kernel headers);
* all the agent components, included the previously built `libbcc` and `system-probe`, are bundled together with omnibus in the legacy `{rpm,deb}-$ARCH` environment.
And, in this last step, omnibus is checking that everything that is shipped inside `/opt/datadog-agent/embedded` can resolve all its symbols in libs that are also shipped inside `/opt/datadog-agent/embedded`.

If, because of the build process inside `bullseye`, `libbcc` was depending on a glibc symbol not available on `wheezy` or `centos:6`, the build would fail. (Unfortunately, I cannot find the GitLab job where I saw that anymore, but I saw this check)

Reply:
👍Thanks! I wasn't sure if omnibus checked each symbol or just the soname. Thanks for taking this into account and for the detailed explanation 🙇